README.md

```diff
@@ -6,12 +6,16 @@
 - `cd snipt`
 - `python3 -m venv ~/.virtualenvs/snipt`
 - `source ~/.virtualenvs/snipt/bin/activate`
+- `From the requirements.txt remove line number 17 and line number 21`
+- `Run the following commands: pip install django-pagination-py3 and pip install psycopg2-binary`
```

Comment:
Added directly to `requirements.txt` in https://github.com/nicksergeant/snipt/commit/3d749b83b2f7a55072d7be384a4e10eabaa5c5ba